draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -239,8 +239,7 @@ This is because the catalog zone consumer does not have the `<unique-N>` identif
 It has to wait for an update of `$NEWCATZ` adding or changing that member zone.
 
 When a catalog zone consumer of catalog zone `$NEWCATZ` receives an update of `$NEWCATZ` which adds or changes a member zone, *and* that consumer had the member zone associated with `$OLDCATZ`, *and* there is an `coo` property of the member zone in `$OLDCATZ` pointing to `$NEWCATS`, *only then* it will reconfigure the member zone with the for `$NEWCATZ` preconfigured settings.
-
-[FIXME: reset state with `epoch` or without property perhaps?]
+All associated state for the zone (such as the zone data, or DNSSEC keys) is in such case reset, unless the `epoch` property (see (#epochproperty)) is supported by the catalog zone consumer and the member zone in both `$OLDCATZ` and `$NEWCATZ` have a matching `epoch` property.
```

Comment:
What does `matching` mean? (What if the epoch in the new catalog is lower?)

Reply:
Matching means equal I suppose... A lower value would mean reset. Do you agree?

Reply:
For the moment I'll change "have a matching `epoch` property" into "have an `epoch` property with an equal value."

Reply:
I agree, I just thought that stating equality explicitly would make it more clear. (Others may have read "matching" as "compatible", and then you get in the business of which ordering is compatible.)